xarray/namedarray/_typing.py

```diff
@@ -55,6 +55,7 @@ def dtype(self) -> _DType_co:
 _ShapeType_co = TypeVar("_ShapeType_co", bound=Any, covariant=True)
 
 _Chunks = tuple[_Shape, ...]
+_NormalizedChunks = tuple[tuple[int, ...], ...]
```

Comment:
I think we can use either or here. I'm not convinced that using TypeVar in _Chunks is the best idea either.